src/components/core/HeatMap.tsx

```diff
@@ -11,7 +10,6 @@ export class HeatMap extends React.Component<IChartProps, null> {
 
     constructor(props: IChartProps) {
         super(props);
-        visualizationIsBetaWarning();
```

Comment:
keep it here, because we want to release SDK 5.2 this week and heat map is not planned to be delivered in 5.2 in final production quality